### PR TITLE
Fix integration tests.

### PR DESCRIPTION
**Pull request by pavithrasv**
- Mock sys.exit mocking to setup
- Fix invalid common machine config
- Remove test that doesn't add coverage
- Remove tpu test as it isn't supported with latest TF version 
- Fix invalid test bucket error

### CODE REVIEW

File: src/python/tensorflow_cloud/core/tests/integration/run_on_script_test.py [hunk 1]
```diff
@@ -21,18 +21,18 @@
 import tensorflow_cloud as tfc
 
 # Following are the env variables available in test infrastructure:
-#
+
 # The staging bucket to use for cloudbuild as well as save the model and data.
-# TEST_BUCKET = os.environ['TEST_BUCKET']
-#
+_TEST_BUCKET = os.environ["TEST_BUCKET"]
```
Comment (ucdmkt) [inline src/python/tensorflow_cloud/core/tests/integration/run_on_script_test.py:26]: This ought to be 'GCS_TEST_BUCKET', per CI configuration

File: src/python/tensorflow_cloud/core/tests/integration/run_on_script_test.py [hunk 2]
```diff
@@ -21,18 +21,18 @@
...
+
 # The project id to use to run tests.
```
Comment (ucdmkt) [inline src/python/tensorflow_cloud/core/tests/integration/run_on_script_test.py:28]: Those seem unused, if they are indeed unused, please remove.

Comment (pavithrasv) [reply src/python/tensorflow_cloud/core/tests/integration/run_on_script_test.py]: Removed them.

File: src/python/tensorflow_cloud/core/tests/integration/run_on_script_test.py (hunk 1 above)
Comment (pavithrasv) [reply src/python/tensorflow_cloud/core/tests/integration/run_on_script_test.py:24]: It is exported as TEST_BUCKET in the CI config.

**Pull request merged**